mothership-ec/cog-mothership-cms
# Feature/social config

#### What does this do?
Moves social handling to CMS

#### How should this be manually tested?
Include 
```
{{
    render(controller('Message:Mothership:CMS::Controller:Module:Social#share', {
        page: page
    }))
}}

{{
    render(controller('Message:Mothership:CMS::Controller:Module:Social#links'))
}}
```
into a view. also, copy the fixture into config and add some data.

Social links should show and link to correct places

#### Related PRs / Issues / Resources?

#### Anything else to add? (Screenshots, background context, etc)
